CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+# Contributing to NEO
+Neo is an open-source project and it depends on its contributors and constant community feedback to implement the features required for a smart economy. You are more than welcome to join us in the development of Neo.  
```

Comment:
@cschuchardt88 ,organize the changes.
The PR is big enough by itself.

Divide in other PRs this file for example, the Security.md...etc...

After a complete organization I will review.

Break the changes. 
One first PR just with var...another with that...etc..

Reply:
yes, but adding `.editorconfig` to the repo will block all `.github` workflows; having them fail for every PR. But I'll figure something out. But i still can't wait 5 years for a PR to get merged either...